incubator/jaeger/templates/agent-ds.yaml

```diff
@@ -37,60 +37,62 @@ spec:
       hostNetwork: true
       {{- end }}
       dnsPolicy: {{ .Values.agent.dnsPolicy }}
-      nodeSelector:
-{{ toYaml .Values.agent.nodeSelector | indent 8 }}
-{{- if .Values.agent.tolerations }}
-      tolerations:
-{{ toYaml .Values.agent.tolerations | indent 8 }}
-{{- end }}
       serviceAccountName: {{ template "jaeger.agent.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.agent.podSecurityContext | nindent 8 }}
```

Comment:
Why do you use `nindent` instead of `ident` here? What is the reason for the empty line at the beginning?

Reply:
Good point.
I added this as boilerplate for helm create command via https://github.com/helm/helm/pull/5919/files I will make subsequent request to remove the unneeded new line! And adjust it here as well.

Reply:
Oh wait no I think I get it. Due to the `-` in  ``{{- toYaml`` which will remove all empty space, without the newline it sill begin pasting directly after `securityContext:`